dataland-keycloak/dataland_theme/login/templates/login.ftl

```diff
@@ -116,7 +116,7 @@
                             class="p-button p-component uppercase text-primary justify-content-center bg-white-alpha-10 ml-4 cursor-pointer"
                             name="join_dataland_button"
                             onclick="location.href='${url.registrationUrl}'">
-                            JOIN DATALAND
+                            CREATE A PREVIEW ACCOUNT
```

Comment:
This is not center-aligned as in the Figma

Reply:
Should be fixed as well